src/WebJobs.Script/Models/ExtensionPackageReference.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.Models
+{
+    public class ExtensionPackageReference
```

Comment:
Can we add some more comments to this class:
- what is Id? (valid chars?) 
- What are the semantics on Version? Can  this be a richer type than just string?